### ISSUE TITLE
Returning with non-zero exitcode when there are compilation errors

### ISSUE THREAD

**Issue by fvpfvpfvp** [c3lang/c3c#688]
As the title says, it is common for a compiler to exit with exitcode 0 when compilation was successful and with a non-zero exitcode when there were compilation errors.
This means that any other program calling c3c.exe will have a robust way of knowing whether compilation was successful, instead of relying on parsing stdout.

**Comment by lerno** [comment]
Can you please verify that this doesn't happen today? As far as I can tell it *does* return 1 on compilation failure.

**Comment by fvpfvpfvp** [comment]
If I compile
```
module test;
import std::io;

fn int main(char[][] argv) 
{
	asdfasdfasdf::println("Hello World!");
	return 0;
}
```
Using 'c3c compile test.c3' in cmd.exe, I get an error due to the 'asdfasdfasdf'. If I then execute 'echo %errorlevel%', it returns 0. This is latest pre-build, version 0.4.6.

**Comment by lerno** [comment]
@fvpfvpfvp This should be fixed in 0.4.7
